lightning/src/onion_message/messenger.rs

```diff
@@ -1414,7 +1425,8 @@ where
 	fn handle_onion_message(&self, peer_node_id: &PublicKey, msg: &OnionMessage) {
 		let logger = WithContext::from(&self.logger, Some(*peer_node_id), None, None);
 		match self.peel_onion_message(msg) {
-			Ok(PeeledOnion::Receive(message, path_id, reply_path)) => {
+			Ok(PeeledOnion::Receive(message, _tlvs, reply_path)) => {
+				let path_id: Option<[u8; 32]> = None;
```

Comment:
We'll want to drop the `path_id` entirely from `Responder` in a separate commit since it is not useful at this level. For background, see https://github.com/lightningdevkit/rust-lightning/pull/3078#discussion_r1636208042.